src/stories/EkoSubtitles.stories.js

```diff
@@ -6,7 +6,8 @@ export default {
     title: 'Example/EkoSubtitles',
     component: EkoSubtitles,
     argTypes: {
-        style: {}
+        style: {},
+        initialVisibility: {}
```

Comment:
Was this supposed to be a boolean value?

Reply:
https://storybook.js.org/docs/react/api/argtypes
I think the argtypes expects an object that describes how a prop should look in the docs part of the storybook. If you don't supply it here, storybook will use the proptypes defined in the component (I believe). It shows up in the storybook just fine, so I imagine it's fine to keep it as an empty object.